### PR TITLE
Fixed TBProfiler to work with Singularity

### PR DESCRIPTION
**Pull request by jwarnn**
Hello; It was brought to my attention that this tool was not working with Singularity. I just updated the PATH variable so that it works with Singularity.  I also updated some of the versions from 4.3 to 4.3.0  so that all documentation matched. Sorry for the extra work for everyone.

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by kapsakcj** [comment]
Thanks for the PR. We really need to fix the existing GHActions workflow to actually run tools (& their test layers) via singularity to prevent these issues in the future. Sorry you've ran into this.

**Comment by kapsakcj** [comment]
I think that did the trick 👍 I have one small suggestion for the PATH though

```bash
# built successfully w docker
$ docker build -t kapsakcj/tbprofiler:4.3.0 tbprofiler/4.3.0/

# push to personal dockerhub repo
$ docker push kapsakcj/tbprofiler:4.3.0

# download & convert to singularity image format
$ singularity build tbprofiler.4.3.0.sif docker://kapsakcj/tbprofiler:4.3.0

# tb-profiler on PATH inside container
$ singularity exec --no-home tbprofiler.4.3.0.sif tb-profiler
usage: tb-profiler {profile,vcf_profile,fasta_profile,lineage,spoligotype,collate,reprofile,reformat,create_db,update_tbdb,batch,list_db,version} ...

TBProfiler pipeline

positional arguments:
  {profile,vcf_profile,fasta_profile,lineage,spoligotype,collate,reprofile,reformat,create_db,update_tbdb,batch,list_db,version}
                        Task to perform
    profile             Run whole profiling pipeline
    vcf_profile         Run profiling pipeline on VCF file. Warning: this assumes that you have good coverage across the genome
    fasta_profile       Run profiling pipeline on Fasta file. Warning: this assumes that this is a good quality assembly which coveres all drug
                        resistance loci
    lineage             Profile only lineage
    spoligotype         Profile spoligotype (experimental feature)
    collate             Collate results form multiple samples together
    reprofile           Reprofile previous results using a new library. The new library must have same targets and the old one.
    reformat            Reformat json results into text or csv
    create_db           Generate the files required to run TBProfiler
    update_tbdb         Pull the latest tbdb library and load
    batch               Run tb-profiler for several samples
    list_db             Output program version and exit
    version             Output program version and exit

# path inside container looks almost perfect
$ singularity shell --no-home tbprofiler.4.3.0.sif
Singularity> echo $PATH
/opt/conda/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
Singularity> which tb-profiler
/opt/conda/bin//tb-profiler

# singularity version I'm using
$ singularity --version
singularity version 3.8.0
```

File: tbprofiler/4.3.0/Dockerfile [hunk 1]
```diff
@@ -37,12 +37,13 @@ ARG MAMBA_DOCKERFILE_ACTIVATE=1
 # Version of database can be confirmed at ./TBProfiler-${TBPROFILER_VER}/db/tbdb.version.json
 RUN tb-profiler update_tbdb
 
-ARG MAMBA_DOCKERFILE_ACTIVATE=1
+ENV MAMBA_DOCKERFILE_ACTIVATE=1
 WORKDIR /data
 
-FROM app as test
+ENV PATH="/opt/conda/bin/:${PATH}"
```
Comment (kapsakcj) [inline tbprofiler/4.3.0/Dockerfile:43]: please remove the `/` after `bin` on line 43

Everything still works and this doesn't really matter, but you can see that the addition to the PATH variable is a bit different than the defaults:
```
Singularity> echo $PATH
/opt/conda/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment (jwarnn) [reply tbprofiler/4.3.0/Dockerfile]: Done

**Review by kapsakcj — APPROVED** [review]

**Comment by jwarnn** [comment]
'/' removed.

**Comment by kapsakcj** [comment]
Thank you! Once the test passes, I'll merge and deploy to dockerhub and quay with the new docker image name & tag `staphb/tbprofiler:4.3.0` and `staphb/tbprofiler:latest`

I'll delete the `:4.3` tag too

**Pull request merged**